platform/locale.hpp

```diff
@@ -10,6 +10,8 @@ struct Locale
   std::string m_language;
   std::string m_country;
   std::string m_currency;
+  std::string m_decimalSeparator;
```

Comment:
Can a decimal separator also be a non-ASCII symbol? Is there a full list somewhere?

Reply:
Yes, according to [this](https://en.wikipedia.org/wiki/Decimal_separator) Wikipedia article, there's also an Arabic decimal separator (U+066B).

Here's a screenshot of the place info panel with locate set to Farsi/Persian, where the decimal and grouping separators are both non-ASCII chars:
![Captura desde 2023-08-17 00-08-14](https://github.com/organicmaps/organicmaps/assets/25888296/cceb989c-00ed-435b-9c64-a33706e18622)